.circleci/config.yml

```diff
@@ -357,7 +376,7 @@ jobs:
     machine:
       enabled: true
       image: ubuntu-2004:202010-01
-      resource_class: large
+      resource_class: xlarge
```

Comment:
this still don't change the docker resource class of build, memcheck and coverage